pages/docs/global-configuration.en-US.md

```diff
@@ -35,6 +35,79 @@ function App () {
 }
 ```
 
+## Nesting Configurations
+
+`SWRConfig` merges the configuration from the parent context.
```

Comment:
```suggestion
`SWRConfig` merges the configuration from the parent context. It can receive either an object or a functional configuration. The functional one receives the parent configuration as argument and returns a new configuration that you can customize it yourself. 

### Object Configuration Example
```